superbench/runner/runner.py

```diff
@@ -84,7 +84,7 @@ def __validate_sb_config(self):    # noqa: C901
                     if 'proc_num' not in mode:
                         self._sb_benchmarks[name].modes[idx].proc_num = 8
                 elif mode.name == 'mpi':
-                    if 'machinefile' not in mode:
+                    if 'mca' not in mode:
```

Comment:
@RyoYang , could you please also add a test case for this?

Reply:
and these 2 bugs in omegaconf upgrade PR seems no relationship with the title/description? please change the title or split the PR

Reply:
Updated.